abi/include/abi/elf.h

```diff
@@ -109,18 +109,18 @@
 /**
  * ELF section types
  */
-#define SHT_NULL      0
```

Comment:
This looks like whitespace changes only, can you please remove it from the PR?

Reply:
Sure, this was originally caused by me adding new SHT macros (for the INIT_ARRAY and FINI_ARRAY sections, which were longer and needed extra alignment, but later section reading was removed from the elf reader). Just to be sure, I can fix this regularly by a new commit, right?

Reply:
Sure.

Reply:
Reverted whitespace changes and tcb_t casting in the latest commit.